.tools/validate.go

```diff
@@ -16,6 +16,12 @@ import (
 var DefaultRules = []ValidateRule{
 	func(c CommitEntry) (vr ValidateResult) {
 		vr.CommitEntry = c
+		if len(strings.Split(c.Parent, " ")) > 1 {
```

Comment:
meh. that would be another separated shell-out, and would then also have to be done for abbreviated parent as well.